controllers/storagecluster/prometheus/localcephrules.yaml

```diff
@@ -22,8 +22,8 @@ spec:
         count(ceph_osd_metadata{job="rook-ceph-mgr"})
       record: job:ceph_osd_metadata:count
     - expr: |
-        count(kube_persistentvolume_info * on (storageclass)  group_left(provisioner) kube_storageclass_info {provisioner=~"(.*rbd.csi.ceph.com)|(.*cephfs.csi.ceph.com)"})
-      record: job:kube_pv:count
+        count(kube_persistentvolume_info * on (storageclass)  group_left(provisioner) kube_storageclass_info {provisioner=~"(.*rbd.csi.ceph.com)|(.*cephfs.csi.ceph.com)|(.*topolvm.cybozu.com)"})
+      record: job:odf_system_pvs:count
```

Comment:
@weirdwiz , can you confirm, whether the name provided `odf_system_pvs` is intended (not `odf_system_pv`)?

Reply:
Ref: https://coreos.slack.com/archives/C0VMT03S5/p1659526714221299?thread_ts=1659505152.682759&cid=C0VMT03S5